openapi/bintable/client.bal

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/url;
+import ballerina/lang.'string;
+
+# This is a generated connector from [BINTable API v1](https://bintable.com/get-api) OpenAPI Specification. 
+# BIN lookup API, the free api service from bintable.com to lookup card information using it's BIN. 
+# The service maintains updated database based on the comunity and other third party services to make sure all BINs in the database are accurate and up to date.
+public isolated client class Client {
+    final http:Client clientEp;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization doesn't require setting the API credentials.
+    # Create a [BINTable Account](https://bintable.com/get-api) and obtain the API key.
+    #
```

Comment:
Shall we add another comment like this?
`    # Some operations may require passing the token as a parameter.`

You can find a similar connector https://github.com/ballerina-platform/ballerinax-openapi-connectors/blob/main/openapi/fraudlabspro.frauddetection/client.bal#L30